### PR TITLE
Adjust CSB cred

### PR DESCRIPTION
**Pull request by loneil**
Set Issuer DID for cred def allowance to schema owner

### CODE REVIEW

**Comment by WadeBarnes** [comment]
The changes look good but the definition of `issuer_did` and `author_did` in the context of the `credential_definition` rules is not well defined in the documentation which leads to confusion.

Created a ticket for this; https://github.com/hyperledger/aries-endorser-service/issues/146

**Review by WadeBarnes — APPROVED** [review]

**Comment by esune** [comment]
@loneil feel free to merge this if/when the configuration is applied to `dev` (I think it is already there, but waiting for confirmation).

**Pull request merged**
